elife/postgresql-appdb.sls

```diff
@@ -18,10 +18,13 @@
 
 # handles permissions on legacy databases
 db-perms-to-rds_superuser:
-{% if db_exists and app_user_exists %}
     cmd.script:
-        - name: salt://elife/scripts/postgresql-appdb-perms.sh
-        - creates: /root/legacy-db-permissions-migrated.flag
+    {% if db_exists and app_user_exists %}
+        - name: salt://elife/scripts/postgresql-appdb-perms-legacy.sh
```

Comment:
so if I understand correctly this terminology
1. `legacy` should run once on all stacks, then we can remove it?
2. We will then leave only `fresh` which will only be run once on the first successful high state, then never again?
3. If the above is true, can we add `TODO` comments saying when to remove `legacy`?

Reply:
all projects that opted in to the postgresql-appdb state should have had the legacy script run already. future projects that decide to opt in will need that script run on them to migrate any permissions. until the appdb work becomes default, and I don't think it should, the legacy script must remain.

Reply:
> until the appdb work becomes default, and I don't think it should, the legacy script must remain.

So can it be named something like `migration-to-appdb` or similar, rather than `legacy` which as a name pushes for its own cleanup?

Reply:
ah, I see your problem now. I wasn't thinking of legacy as legacy. I was thinking of legacy as legacy. http://www.dictionary.com/browse/legacy

I'll rename it so it's clearer